docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -136,46 +245,209 @@ To retrieve the metadata and playback information on the client side:
+   To retrieve the playback information from the server, use `mc_client_get_server_playback_info()`:
 
    ```
    mc_playback_h playback = NULL;
    mc_playback_states_e playback_state;
+   unsigned long long position = 0;
+   char *playlist_name = NULL;
+   char *index = NULL;
+   mc_content_type_e content_type;
+   mc_content_age_rating_e age_rating;
+
 
    ret = mc_client_get_server_playback_info(g_client_h, server_name, &playback);
 
    ret = mc_client_get_playback_state(playback, &playback_state);
-   dlog_print(DLOG_DEBUG, LOG_TAG, "Playback State: %d\n", playback_state);
-   ```
 
-   The `mc_client_get_playback_state()` retrieves the playback state from the playback information returned by the `mc_client_get_server_playback_info()`.
+   ret = mc_client_get_playback_position(playback, &position);
+
+   ret = mc_client_get_playlist_item_index(playback, &index);	//only Tizen 4.0 for Mobile
+
+   ret = mc_client_get_playlist_item_info(playback, &playlist_name, &index);	//since Tizen 5.0
+
+   ret = mc_client_get_playback_content_type(playback, &content_type);	//since Tizen 5.0
 
-4. Destroy the media controller client handle using the `mc_client_destroy()`, when media controller client handle is no longer needed:
+   ret = mc_client_get_age_rating(playback, &age_rating);	//since Tizen 5.0
+   ```
+
+4. Destroy the media controller client handle using `mc_client_destroy()`, when media controller client handle is no longer needed:
 
    ```
    mc_client_destroy(g_client_h);
    ```
 
 
+## Updating and Retrieving Metadata
+
+To update the metadata information on the server side, follow these steps:
+
+1. Create the media controller server handle using `mc_server_create()`:
+
+   ```
+   ret = mc_server_create(&g_server_h);
+   ```
+
+2. Set the metadata to be updated using `mc_server_set_metadata()` and then update the metadata using `mc_server_update_metadata()`.
+
+   ```
+   mc_server_set_metadata(g_server_h, MC_META_MEDIA_TITLE, "title_1");
+   mc_server_set_metadata(g_server_h, MC_META_MEDIA_ARTIST, "artist_1");
+   mc_server_set_metadata(g_server_h, MC_META_MEDIA_ALBUM, "album_1");
+   mc_server_set_metadata(g_server_h, MC_META_MEDIA_AUTHOR, "author_1");
+   mc_server_set_metadata(g_server_h, MC_META_MEDIA_GENRE, "genre_1");
+   mc_server_set_metadata(g_server_h, MC_META_MEDIA_DURATION, "duration_1");
+   mc_server_set_metadata(g_server_h, MC_META_MEDIA_DATE, "date_1");
+   mc_server_set_metadata(g_server_h, MC_META_MEDIA_COPYRIGHT, "copyright_1");
+   mc_server_set_metadata(g_server_h, MC_META_MEDIA_DESCRIPTION, "desc_1");
+   mc_server_set_metadata(g_server_h, MC_META_MEDIA_TRACK_NUM, "tracknum_1");
+   mc_server_set_metadata(g_server_h, MC_META_MEDIA_PICTURE, "picture_1");
+   
+   ret = mc_server_update_metadata(g_server_h);
+   ```
+   
+   You must encode the values for the MC_META_MEDIA_SEASON, MC_META_MEDIA_EPISODE, MC_META_MEDIA_RESOLUTION. 
```

Comment:
@haejeongkim 
Kindly change to: 

You must encode the values for MC_META_MEDIA_SEASON, MC_META_MEDIA_EPISODE, and MC_META_MEDIA_RESOLUTION.